addons/account/controllers/portal.py

```diff
@@ -57,7 +57,7 @@ def _get_overdue_invoices_domain(self, partner_id=None):
         return [
             ('state', 'not in', ('cancel', 'draft')),
             ('move_type', 'in', ('out_invoice', 'out_receipt')),
-            ('payment_state', 'not in', ('in_payment', 'paid')),
+            ('payment_state', 'not in', ('in_payment', 'paid', 'reversed')),
```

Comment:
Does it make sense for `blocked` or `invoicing_legacy`?

Reply:
I guess no